openapi/mx_platform_api_beta.yml

```diff
@@ -0,0 +1,3795 @@
+          content:
+            application/vnd.mx.api.v1+json:
+              schema:
+                "$ref": "#/components/schemas/TransactionResponseBody"
+          description: OK
+      summary: Read transaction
+      tags:
+      - Transaction
+    put:
+      description: Use this endpoint to update the `description` of a specific transaction
+        according to its unique GUID.
+      operationId: UpdateTransaction
+      parameters:
+      - description: The unique id for a `transaction`.
+        example: TRN-810828b0-5210-4878-9bd3-f4ce514f90c4
+        in: path
+        name: transaction_guid
+        required: true
+        schema:
+          type: string
+      - description: The unique id for a `user`.
+        example: USR-fa7537f3-48aa-a683-a02a-b18940482f54
+        in: path
+        name: user_guid
+        required: true
+        schema:
+          type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              "$ref": "#/components/schemas/TransactionUpdateRequestBody"
+        description: Transaction object to be updated with a new description
+        required: true
+      responses:
+        '200':
+          content:
+            application/vnd.mx.api.v1+json:
+              schema:
+                "$ref": "#/components/schemas/TransactionResponseBody"
+          description: OK
+      summary: Update transaction
+      tags:
+      - Transaction
+  "/users/{user_guid}/widget_urls":
+    post:
+      description: This endpoint allows partners to get a URL by passing the `widget_type`
+        in the request body, as well as configuring it in several different ways.
+        In the case of Connect, that means setting the `widget_type` to `connect_widget`.
+        Partners may also pass an optional `Accept-Language` header as well as a number
+        of configuration options. Note that this is a `POST` request.
+      operationId: RequestWidgetURL
+      parameters:
+      - description: The unique id for a `user`.
+        example: USR-fa7537f3-48aa-a683-a02a-b18940482f54
+        in: path
+        name: user_guid
+        required: true
+        schema:
+          type: string
+      - description: The desired language of the widget.
+        example: en-US
+        in: header
+        name: Accept-Language
+        schema:
+          type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              "$ref": "#/components/schemas/WidgetRequestBody"
+        description: The widget url configuration options.
+        required: true
+      responses:
+        '200':
+          content:
+            application/vnd.mx.api.v1+json:
+              schema:
+                "$ref": "#/components/schemas/WidgetResponseBody"
+          description: OK
+      summary: Request a widget URL
+      tags:
+      - Widget
+security:
+- basicAuth: []
+servers:
+- url: https://api.mx.com
+- url: https://int-api.mx.com
+tags:
```

Comment:
Do we still need these tags listed here?

Reply:
We may need to reduce this list down to one tag. If I remember correctly the openapi generator will treat the different tags as separate "apis" when generating a library.

In this cut I was focusing on the request and response bodies. Once this is PR is approved, I'll run it through the openapi generator and check on the tags.